src/Components/ScrivitoExtensions/SocialCardsTab.js

```diff
@@ -36,79 +36,131 @@ Scrivito.registerComponent("SocialCardsTab", ({ obj }) => (
 
-const TwitterInput = Scrivito.connect(({ obj }) => (
-  <div>
-    <div className="scrivito_detail_label">
-      <span className="headline">Twitter</span>
-    </div>
-    <div className="scrivito_detail_label">
-      <span>Creator</span>
-    </div>
-    <Scrivito.ContentTag
-      content={obj}
-      attribute="tcCreator"
-      className="input"
-    />
-    <div className="scrivito_notice_body">
-      Twitter handle of the tweet creator. Start with {"@"}
-    </div>
-    <div className="scrivito_detail_label">
-      <span>Image</span>
-    </div>
-    <Scrivito.ImageTag
-      content={obj}
-      attribute="tcImage"
-      className="seo_card_img"
-    />
-    <div className="scrivito_notice_body">Add or replace the image here.</div>
-    <div className="scrivito_detail_label">
-      <span>Title</span>
-    </div>
-    <Scrivito.ContentTag content={obj} attribute="tcTitle" className="input" />
-    <div className="scrivito_detail_label">
-      <span>Description</span>
-    </div>
-    <Scrivito.ContentTag
-      content={obj}
-      attribute="tcDescription"
-      className="input"
-    />
-    <div className="scrivito_notice_body">Limit to 140 characters</div>
-  </div>
-));
+const TwitterInput = Scrivito.connect(({ obj }) => {
+  const validationsForCreator = Scrivito.validationResultsFor(obj, "tcCreator");
+  const creatorValidationClass = validationsClassName(validationsForCreator);
 
-const FacebookInput = Scrivito.connect(({ obj }) => (
-  <div>
-    <div className="scrivito_detail_label">
-      <span className="headline">Facebook</span>
-    </div>
-    <div className="scrivito_detail_label">
-      <span>Image</span>
-    </div>
-    <Scrivito.ImageTag
-      content={obj}
-      attribute="ogImage"
-      className="seo_card_img"
-    />
-    <div className="scrivito_notice_body">Add or replace the image here.</div>
-    <div className="scrivito_detail_label">
-      <span>Title</span>
-    </div>
-    <Scrivito.ContentTag content={obj} attribute="ogTitle" className="input" />
-    <div className="scrivito_notice_body">Add a catchy title for the post.</div>
-    <div className="scrivito_detail_label">
-      <span>Description</span>
+  const validationsForDescription = Scrivito.validationResultsFor(
+    obj,
+    "tcDescription"
+  );
+  const descriptionValidationClass = validationsClassName(
+    validationsForDescription
+  );
+
+  return (
+    <div>
+      <div className="scrivito_detail_label">
+        <span className="headline">Twitter</span>
+      </div>
+      <div className={`scrivito_detail_label ${creatorValidationClass}`}>
+        {creatorValidationClass && (
+          <i className="scrivito_icon scrivito_icon_error"></i>
+        )}
+        <span>Creator</span>
+      </div>
+      <Scrivito.ContentTag
+        content={obj}
+        attribute="tcCreator"
+        className={`input ${creatorValidationClass}`}
+      />
+      <div>
+        <ValidationMessages validations={validationsForCreator} />
```

Comment:
Please remove the upper `div`. `ValidationMessages` is already rendering a surrounding `div`.